packages/asc-ui/src/components/Input/Input.tsx

```diff
@@ -23,62 +23,63 @@ export interface InputProps extends InputMethods {
   error?: string
 }
 
-export interface Shared {}
+const Input = React.forwardRef<
+  HTMLInputElement,
+  InputProps & React.HTMLAttributes<HTMLInputElement>
+>(
+  (
+    { blurOnEscape, focusOnRender, value, error, ...props }: InputProps,
```

Comment:
If i'm not mistaken, you can remove the  :InputProps and the :any, as these should already be typed

Reply:
dankjewel

Reply:
gedaan